test/std-test.cc

```diff
@@ -12,6 +12,7 @@
 
 #include "fmt/os.h"  // fmt::system_category
 #include "fmt/ranges.h"
+#include "fmt/xchar.h"
```

Comment:
std-test shouldn't include xchar.h. I you want to test wide strings, move the test to xchar-test.